docker-compose.yml

```diff
@@ -3,9 +3,11 @@ version: "3"
 services:
   pmm-server:
     container_name: pmm-server
-    image: perconalab/pmm-server:dev-latest
+    image: perconalab/pmm-server:dev-container
     volumes:
+      - "./:/workspace"
       - "../grafana/public:/usr/share/grafana/public"
```

Comment:
Please change to - "./public:/usr/share/grafana/public"

Reply:
It was made with intention. You can easily copy-paste this to PMM docker compose. And it will work.
But it will not work if you renamed directory while clonning.